src/igvfd/audit/measurement_set.py

```diff
@@ -182,3 +182,47 @@ def audit_preferred_assay_title(value, system):
             f'which is not an expected preferred assay title for this assay term.'
         )
         yield AuditFailure('inconsistent assay metadata', detail, level='WARNING')
+
+
+@audit_checker('MeasurementSet', frame='object')
+def audit_inconsistent_institutional_certification(value, system):
+    '''
+        audit_detail: Measurement sets for mapping assays are expected to link to samples covered by an institutional certificate issued to a matching lab and award.
+        audit_category: inconsistent institutional certification
+        audit_levels: ERROR
+    '''
+    # Characterization assays do not need to be audited.
+    characterization_assays = [
```

Comment:
if we are going for assay_temr objects, I think it would be more future proof to switch to IDs rather than terms, even though the code will be less "readable" the term could change in ontology, the ID should be more stable